.github/workflows/documentation.yaml

```diff
@@ -19,7 +19,7 @@ concurrency:
 jobs:
   check_changes:
     name: Deduce required tests from code changes
-    runs-on: ubuntu-24.04
```

Comment:
Where did this came from?

Reply:
I added that to fix the documentation workflow after https://github.com/cilium/cilium/pull/35092 bumped it (and broke the workflow)